dotcom-rendering/src/layouts/FullPageInteractiveLayout.tsx

```diff
@@ -164,11 +166,14 @@ const NavHeader = ({ article, NAV, format }: Props) => {
 					element="nav"
 				>
 					<Nav
-						format={{
-							display: format.display,
-							design: format.design,
-							theme: getCurrentPillar(article),
-						}}
+						isImmersive={
+							format.display === ArticleDisplay.Immersive
+						}
+						displayRoundel={
+							format.display === ArticleDisplay.Immersive ||
+							format.theme === ArticleSpecial.Labs
+						}
```

Comment:
Given that these rules are now repeated in a few places, I wonder if they should be captured at the point where we derive `NavType`? So `isImmersive` and `displayRoundel` become properties of that type, and are derived from `ArticleFormat` for articles?

Not a blocker, but curious what you think?

Reply:
yeah I went back and forth on this one - one could argue these should just be set to "true" here anyway, as we're in the immersive layout so format should always be display immersive

would it be ok to do this in another PR? 🙏

Reply:
> would it be ok to do this in another PR?
👍